src/amo/server/WebpackAssetsFontsPlugin.js

```diff
@@ -0,0 +1,41 @@
+import fs from 'fs-extra';
+
+// This is a webpack plugin to add .woff2 fonts in webpack-assets.json that
+// webpack-isomorphic-tools generates. We can then use `assets` to reference
+// these fonts in server side rendered code to preload some font subsets, like
+// we can in development mode.
+
+export default class WebpackAssetsFontsPlugin {
+  constructor({ webpackAssetsFileName } = {}) {
+    if (!webpackAssetsFileName) {
+      throw new Error('The webpackAssetsFileName parameter cannot be empty');
```

Comment:
I think it'd be slightly better to use the schema approach, which is the recommended way to validate webpack plugin options: https://webpack.js.org/contribute/writing-a-plugin/#basic-plugin-architecture

Reply:
I got the idea from our own `sriDataPlugin.js` plugin :)

Reply:
> I got the idea from our own `sriDataPlugin.js` plugin :)

yeah I was pretty sure. This plugin was written for webpack 1 I think.